.github/workflows/autobump.yml

```diff
@@ -40,21 +43,20 @@ jobs:
         with:
           username: ${{ (github.event_name == 'workflow_dispatch' && github.actor) || 'BrewTestBot' }}
 
-      - name: Get list of autobump formulae
-        id: autobump
-        run: echo "autobump_list=$(xargs < "$(brew --repo homebrew/core)"/.github/autobump.txt)" >> "$GITHUB_OUTPUT"
-
       - name: Set up commit signing
         uses: Homebrew/actions/setup-commit-signing@master
         with:
           signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
 
       - name: Bump formulae
-        uses: Homebrew/actions/bump-packages@master
```

Comment:
If we do this: is this action used anywhere in the Homebrew org any more? If not: it might be nice to fold the below into that action somehow.

Reply:
Currently used in Homebrew/cask, I think. I'll try to fold these changes into the action anyway.